src/merge.c

```diff
@@ -1711,8 +1732,13 @@ int git_merge__iterators(
 
 	git_vector_foreach(&changes, i, conflict) {
 		int resolved = 0;
+		int file_favor = opts.file_favor;
+
+		/* Check for merge options in .gitattributes */
+		if ((error = git_merge__lookup_file_favor(repo, conflict->our_entry.path, &file_favor) < 0))
```

Comment:
Good question. It was an arbitrary decision because for a given conflict, the presumption was that `our_entry` and `their_entry` should have the same path name. I'm not totally sure if that is correct. Reading `ll-merge.c` from git, I'm not quite sure what it is doing yet.